packages/zipkin/src/serializeSpan.js

```diff
@@ -1,14 +1,16 @@
-const {TBufferedTransport, TBinaryProtocol} = require('thrift');
 const {fromByteArray: base64encode} = require('base64-js');
 
-let serialized;
-const transport = new TBufferedTransport(null, res => {
-  serialized = res;
-});
+module.exports = function serializeSpan(span, format = 'base64') {
+  // eslint-disable-next-line global-require
+  const {TBufferedTransport, TBinaryProtocol} = require.call(null, 'thrift');
```

Comment:
serialize span is always called when you want to transport a span, right? This means as a non-node user you will require thrift, which will fail or am I mistaken?